{"status": "resolved", "priority": "bug", "title": "Impossible to Import a data snapshot: \"Multiple fixtures named '2012-08-12' in absolute path. Aborting.\"", "milestone": "", "superceder": "", "nosylist": ["e2jk, paulproteus"], "assigned": "", "waitingon": "", "keywords": [], "id": "757", "files": [], "history": [{"message": "Indeed, I had both the json and json.gz files in there. The wiki instructed to\nungzip, then load the unzipped file, so I've updated the wiki to specify that it\ndoesn't need to be uncompressed:\n<a href=\"https://openhatch.org/wiki/Importing_a_data_snapshot#How_to_use_a_snapshot\">https://openhatch.org/wiki/Importing_a_data_snapshot#How_to_use_a_snapshot</a>\n\nIssue closed!\n   \n", "author": "e2jk"}, {"message": "Hi e2jk,\n\nIs it possible you have two files in that directory called 2012-08-12* -- in \nparticular, maybe the json and json.gz file are both there?\n   \n", "author": "paulproteus"}, {"message": "I just cloned Openhatch (commit 08218b8ded), ran `python manage.py syncdb --\nnoinput` then `python manage.py migrate`, and following the instruction at <a href=\"https://openhatch.org/wiki/Importing_a_data_snapshot\">https://openhatch.org/wiki/Importing_a_data_snapshot</a> I downloaded the latest \navailable snapshot <a href=\"http://inside.openhatch.org/snapshots/2012-08-12.json.gz,\">http://inside.openhatch.org/snapshots/2012-08-12.json.gz,</a> \nungzipped it and executed `python manage.py loaddata 2012-08-12.json`\n\nAfter running for about 5-10 minutes, the process stops with this error message:\n\nMultiple fixtures named '2012-08-12' in absolute path. Aborting.\n\nAs a result my local copy of Openhatch has no data (users, projects, etc.)\n   \n", "author": "e2jk"}]}